doc/quadrature/naive_monte_carlo.qbk

```diff
@@ -12,7 +12,8 @@ LICENSE_1_0.txt or copy at http://www.boost.org/LICENSE_1_0.txt)
     #include <boost/math/quadrature/naive_monte_carlo.hpp>
     namespace boost { namespace math { namespace quadrature {
 
-    template<class Real, class F, class RNG = std::mt19937_64, class Policy = boost::math::policies::policy<>>
+    template<class Real, class F, class RNG = std::mt19937_64, class Policy = boost::math::policies::policy<>, 
```

Comment:
We really should get PCG into boost.random. Mersenne twister is just an awful RNG>

Reply:
It seems difficult to get even trivial changes into boost.random...